src/components/MapView/Analyser/index.tsx

```diff
@@ -0,0 +1,317 @@
+/* eslint-disable no-console */
+import React, { useMemo, useState } from 'react';
+import {
+  Button,
+  createStyles,
+  FormControl,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import { Assessment, ArrowDropDown } from '@material-ui/icons';
+import { useSelector } from 'react-redux';
+import { map, find } from 'lodash';
+import bbox from '@turf/bbox';
+import {
+  getBoundaryLayerSingleton,
+  LayerDefinitions,
+} from '../../../config/utils';
+import {
+  BoundaryLayerProps,
+  NSOLayerProps,
+  WMSLayerProps,
+} from '../../../config/types';
+import { ApiData, fetchApiData } from '../../../utils/flask-api-utils';
+import { getWCSLayerUrl } from '../../../context/layers/wms';
+import { LayerData } from '../../../context/layers/layer-data';
+import { layerDataSelector } from '../../../context/mapStateSlice';
+import { Extent } from '../Layers/raster-utils';
+import { availableDatesSelector } from '../../../context/serverStateSlice';
+
+type StatisticType = 'mean' | 'median';
+
+const layers = Object.values(LayerDefinitions);
+const baselineLayers = layers.filter(
+  (layer): layer is NSOLayerProps => layer.type === 'nso',
+);
+const hazardLayers = layers.filter(
+  (layer): layer is WMSLayerProps => layer.type === 'wms',
+);
+const statistics: StatisticType[] = ['mean', 'median'];
```

Comment:
Why can't we use AggregationOperations?

Reply:
I believe we thought the Analysis UI would have more statistic options than supported based off the UI design in #86. I'll fix it next commit :+1:

Reply:
Fair enough! Maybe create a new type in the flask-utils and use that?

Reply:
Since the Impact and Analysis logic is tied closely together, I'll keep it on AggregationOperations for simplicity's sake. If @dubwalla did in fact imply custom statistic types from his design in #86 then we can look at switching it back to this old system :+1: